dist/arNFT.js

```diff
@@ -202,7 +202,14 @@ function start (container, markerUrl, video, input_width, input_height, canvas_d
 
     renderer.setSize(sw, sh);
 
-    worker = new Worker(configData.workerUrl);
+    // create a Worker to handle loading of NFT marker and tracking of it
+    var workerBlob = new Blob(
```

Comment:
Probably Blob is causing the issue when it is minified? Need to investigate on this...

Reply:
see #10